test/metabase/query_processor_test/expressions_test.clj

```diff
@@ -39,6 +39,20 @@
          :limit       3
          :order-by    [[:asc $id]]}))))
 
+;; Make sure FLOATING POINT division is done when dividing by expressions/fields
+(datasets/expect-with-drivers (qp.test/non-timeseries-drivers-with-feature :expressions)
+  [[0.6]
+   [0.5]
+   [0.5]]
+  (qp.test/format-rows-by [double]
```

Comment:
If you need to round this because some drivers are doing weird stuff you can do `format-rows-by [1.0]` now for example which will round to the nearest `1` decimal place